### PR TITLE
implemented healthcheck in quickstart script

### PR DESCRIPTION
**Pull request by talesprates**
now quickstart performs healthcheck during fifa champs start script

### CODE REVIEW

File: api/routes.js [hunk 1]
```diff
@@ -308,4 +308,8 @@ exports.set = function(app, mongo, log) {
     app.get(URL_TEAM_PICK, (request, response) => {
         response.send(JSON.stringify(teamsJSON, null, 2));
     });
+
+    app.get('/api/healthcheck', (request, response) => {
```
Comment (cristiadu) [inline api/routes.js:312]: Try to not hardcode stuff in here, use the same terminology the other routes have.

File: bash_resources/healthcheck.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
+function stop_fifachamps () {
```
Comment (cristiadu) [inline bash_resources/healthcheck.sh:1]: This is not part of the healthcheck, move it somewhere else.

File: bash_resources/healthcheck.sh [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+  echo 'mongo healthcheck failed'
+  stop_fifachamps
+  exit 1
+}
+
+function api_healthcheck() {
+  for i in `seq 1 5`; do
+    echo 'performing api healthcheck '${i}'/5'
+    curl -sL -X GET localhost:5001/api/healthcheck > /dev/null
```
Comment (cristiadu) [inline bash_resources/healthcheck.sh:26]: I would like for URLs to not be hardcoded so we could configure them on different ports, I know that's not the way it is right now, so this is not an issue by itself, but on next interactions, try to think about that as well.

File: bash_resources/healthcheck.sh [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+  echo 'api healthcheck failed'
+  stop_fifachamps
+  exit 1
+}
+
+function webapp_healthcheck() {
+  for i in `seq 1 5`; do
+    echo 'performing webapp healthcheck '${i}'/5'
+    curl -sL -X GET localhost:4200 > /dev/null
```
Comment (cristiadu) [inline bash_resources/healthcheck.sh:42]: Same as the other comment, having this hardcoded port is not ideal. But we can keep this way for now.

**Review by cristiadu — CHANGES_REQUESTED** [review]
Great work! Just some small comments, mainly related to hardcoded stuff.

**Review by cristiadu — CHANGES_REQUESTED** [review]
Great work! Just some small comments, mainly related to hardcoded stuff.

**Review by kiosia — APPROVED** [review]
Apart from @cristiadu said, approved! :)

File: api/routes.js [hunk 5]
```diff
@@ -308,4 +308,8 @@ exports.set = function(app, mongo, log) {
...
+        response.send({ status : 'OK' });
```
Comment (alandoni) [inline api/routes.js:313]: you could also make a request to database before sending OK, some "find" command is enough to make sure everything is working

**Review by alandoni — CHANGES_REQUESTED** [review]

**Pull request closed without merging**